kafka-ui-api/src/test/java/com/provectus/kafka/ui/serde/SimpleRecordSerDeTest.java

```diff
@@ -3,27 +3,19 @@
 import static com.provectus.kafka.ui.serde.RecordSerDe.DeserializedKeyValue;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.provectus.kafka.ui.model.KafkaCluster;
 import com.provectus.kafka.ui.serde.schemaregistry.MessageFormat;
-import com.provectus.kafka.ui.serde.schemaregistry.SchemaRegistryAwareRecordSerDe;
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 import org.apache.kafka.common.utils.Bytes;
 import org.junit.jupiter.api.Test;
 
-class SchemaRegistryRecordDeserializerTest {
+class SimpleRecordSerDeTest {
```

Comment:
How did schema registry test became a simple one? 🤔

Reply:
this test was initially named wrong - it didn't test SchemaRegistryAwareRecordSerDe behaviour, but only tests fallback logic that is more applicable for SimpleRecordSerDe.  I will add separate tests for SchemaRegistryAwareRecordSerDe when PR will be approved